src/pages/public_registration/PublicRegistrationContributors.tsx

```diff
@@ -133,7 +145,20 @@ const ContributorsRow = ({ contributors, distinctUnits, hiddenCount }: Contribut
           .filter((affiliationIndex) => affiliationIndex)
           .sort();
 
-        const showRole = contributor.role.type !== ContributorRole.Creator;
+        const hasValidRole = relevantRoles.includes(contributor.role.type);
+
+        const showRole =
+          contributor.role.type !== ContributorRole.Creator || !relevantRoles.includes(ContributorRole.Creator);
```

Comment:
Denne er litt tunglest, så mulig vi burde gå for dette i stedet? Enklere å skjønne hva dette faktisk er da?
```suggestion
        const showRole = relevantRoles.includes(ContributorRole.Creator)
          ? contributor.role.type !== ContributorRole.Creator
          : true;
```

Reply:
Enig